crates/figma_import/src/document.rs

```diff
@@ -153,18 +157,23 @@ impl Document {
     pub fn new(
         api_key: &str,
         document_id: String,
+        version_id: String,
         proxy_config: &ProxyConfig,
         image_session: Option<ImageContextSession>,
     ) -> Result<Document, Error> {
         // Fetch the document...
-        let document_url = format!(
+        let mut document_url = format!(
             "{}{}?plugin_data=shared&geometry=paths&branch_data=true",
             BASE_FILE_URL, document_id,
         );
+        if version_id.is_empty() == false {
+            document_url.push_str("&version=");
+            document_url.push_str(&version_id);
+        }
         let document_root: FileResponse =
             serde_json::from_str(http_fetch(api_key, document_url, proxy_config)?.as_str())?;
 
-        // ...and the mapping from imageRef to URL
+        // ...and the mapping from imageRef to URL. It returns images from all versions.
```

Comment:
Is this just an old comment where the beginning got cut off?  So if this request returns all images, do we just possibly store some extra images that aren't used?

Reply:
Yeah. I didn't make changes here and just added a comment.

Actually [this api](https://www.figma.com/developers/api#get-image-fills-endpoint) would return images from all branches and all versions.